### PR TITLE
update for joblistwork.inc.php

### PR DESCRIPTION
**Pull request by michaelmeganet**

### CODE REVIEW

**Comment by michaelmeganet** [comment]
Change from inherit to independent Class

File: joblistwork.inc.php [hunk 1]
```diff
@@ -550,9 +550,9 @@
     Public jlwsid; 
 
     //later put properties here
-    Public function __construct($jobcode,$cuttingtype, $processcode, $totalquantity, $jobOutputList) {
+    Public function __construct($jobcode) {
 
-        parent::__construct($jobcode, $cuttingtype, $processcode, $totalquantity, $jobOutputList);
+        parent::__construct($jobcodet);
```
Comment (Thraxelon) [inline joblistwork.inc.php:555]: This must be removed as well

**Review by Thraxelon — COMMENTED** [review]
Any of the extended parent functions must be removed,

File: joblistwork.inc.php (hunk 1 above)
Comment (michaelmeganet) [reply joblistwork.inc.php:555]: have to remove the whole line of parent::__construct($jobcodet);

**Pull request merged**